tests/integration/utils.ts

```diff
@@ -155,7 +155,7 @@ export const nftNameCType = CType.fromProperties('NFT collection name', {
 // Submits resolving when IS_IN_BLOCK
 export async function submitTx(
   extrinsic: SubmittableExtrinsic,
-  submitter: KeyringPair,
+  submitter: KeyringPair | Blockchain.TransactionSigner,
```

Comment:
Is this import from Blockchain can't we just import `TransactionSigner`?

Reply:
Yeah that's where it is currently defined